docs/pages/guides/webhooks.mdx

```diff
@@ -10,24 +10,23 @@ you can react to Liveblocks events as they happen. Now that our platform is
 observable, you can use webhooks to reduce development time and the need for
 polling.
 
+## Why Webhooks
+Webhooks connect two applications and enable sending data asynchronously through event subscription and HTTP requests.  Using webhooks with Liveblocks decreases the complexity of managing and creating cron jobs which reach out to our API endpoints and poll. Using webhooks make a platform observable because updates between applications are immediately available.
```

Comment:
```suggestion
Webhooks connect two applications and enable sending data asynchronously through event subscription and HTTP requests. Using webhooks with Liveblocks decreases the complexity of managing and creating cron jobs which reach out to our API endpoints and poll. Using webhooks make a platform observable because updates between applications are immediately available.
```

Reply:
Why don’t we give a few examples to inspire a less technical person?
Here's an example:
> For instance, you could imagine a scenario where you must save your room storage to your database once all users have left it. Our webhooks mechanism will notify your system every time someone leaves the room.

Reply:
sure that makes sense, will add example (or 2)